pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -314,7 +374,8 @@ class CreateVM extends React.Component {
                     <Select.SelectEntry data={URL_SOURCE} key={URL_SOURCE}>{_("URL")}</Select.SelectEntry>
                     { this.props.providerName == 'LibvirtDBus' &&
                     <Select.SelectEntry data={PXE_SOURCE} key={PXE_SOURCE}>{_("Network Boot (PXE)")}</Select.SelectEntry> }
-                    <Select.SelectEntry data={EXISTING_DISK_IMAGE_SOURCE} key={EXISTING_DISK_IMAGE_SOURCE}>{_("Existing Disk Image")}</Select.SelectEntry>
+                    { !this.state.chooseStorage &&
```

Comment:
The logic makes sense more vise versa.
I you chose existing volumes for disk, don't show the choose storage button.. Here is someone wants to change the installation mode to choose existing disk, he has to untick the choose storage checkbox.